dll/win32/profmap/CMakeLists.txt

```diff
@@ -0,0 +1,13 @@
+
+spec2def(profmap.dll profmap.spec)
+
+list(APPEND SOURCE
+    profmap_main.c
```

Comment:
That's not a Wine dll, so you can use instead some other name, like just "profmap.c" or "main.c".

Reply:
OK. Fixed.